### PR TITLE
Vlad's test branch

### PR DESCRIPTION
**Pull request by vmyronov**
Attempting to add new test

### CODE REVIEW

File: api/tests/pokemon.test.js [hunk 1]
```diff
@@ -47,5 +48,11 @@ describe('GET /pokemon', () => {
   it('should accept a limit as an integer', async () => {
     const pokemon = await Posts.getPokemans(path, '40')
     expect(pokemon.body.results.length).toBe(40)
+  }) 
+  // Tried to test the "Count" key's value ->
+  it('Check the count', async () => {
+    path = 'pokemon'
```
Comment (jonyet) [inline api/tests/pokemon.test.js]: path = pokemon is already defined in the `beforeEach` at the top of this test block (L7) -- this effectively sets the path to "pokemon" as the default for every test (every `it` block). you can remove L54 entirely

File: api/tests/pokemon.test.js [hunk 2]
```diff
@@ -47,5 +48,11 @@ describe('GET /pokemon', () => {
...
+    expect(path.body.count).toEqual(1154)
```
Comment (jonyet) [inline api/tests/pokemon.test.js]: every test (`it` block) is a closed block of code, meaning the only thing defined already here is the `path` from `beforeEach`. you need to include `const pokemon = await Posts.getPokemans(path)` to populate the pokemon variable with the api response, then you can do your assertion against that

File: api/tests/pokemon.test.js (hunk 1 above)
Comment (vmyronov) [reply api/tests/pokemon.test.js]: Got it, thanks!

File: api/tests/pokemon.test.js (hunk 2 above)
Comment (vmyronov) [reply api/tests/pokemon.test.js]: Replaced line 54 with the `const pokemon = await Posts.getPokemans(path)`, now it works fine, thanks!

File: api/tests/pokemon.test.js [hunk 3]
```diff
@@ -51,8 +51,13 @@ describe('GET /pokemon', () => {
   }) 
   // Tried to test the "Count" key's value ->
   it('Check the count', async () => {
-    path = 'pokemon'
-    expect(path.body.count).toEqual(1154)
-    console.log(path.body.count)
+    const pokemon = await Posts.getPokemans(path)
```
Comment (vmyronov) [inline api/tests/pokemon.test.js:54]: Replaced the line with the proper path defining variable

File: api/tests/pokemon.test.js [hunk 4]
```diff
@@ -47,5 +48,16 @@ describe('GET /pokemon', () => {
   it('should accept a limit as an integer', async () => {
     const pokemon = await Posts.getPokemans(path, '40')
     expect(pokemon.body.results.length).toBe(40)
+  }) 
+  // Tried to test the "Count" key's value ->
+  it('Check the count', async () => {
```
Comment (jonyet) [inline api/tests/pokemon.test.js]: let's follow the same patterns as the other test descriptions and be more clear with our naming -- e.g: `it('should have a total pokemon count of 1154')`

File: api/tests/pokemon.test.js [hunk 5]
```diff
@@ -47,5 +48,16 @@ describe('GET /pokemon', () => {
   it('should accept a limit as an integer', async () => {
     const pokemon = await Posts.getPokemans(path, '40')
     expect(pokemon.body.results.length).toBe(40)
+  }) 
+  // Tried to test the "Count" key's value ->
```
Comment (jonyet) [inline api/tests/pokemon.test.js]: as you see from the below feedback, we should strive to have our code be as naturally readable as possible, such that code comments are unnecessary :)

**Comment by jonyet** [comment]
can you also hop in and remove all the `console.log` logging please?

**Pull request closed without merging**